src/Workspaces/CSharp/Portable/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -205,15 +197,21 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
             }
 
             // For spacing delimiters - after comma
-            if ((previousToken.IsCommaInArgumentOrParameterList() && currentKind != SyntaxKind.OmittedTypeArgumentToken) ||
-                previousToken.IsCommaInInitializerExpression())
+            if ((previousToken.IsCommaInArgumentOrParameterList() && currentKind != SyntaxKind.OmittedTypeArgumentToken)
+                || previousToken.IsCommaInInitializerExpression()
+                || (previousKind == SyntaxKind.CommaToken
```

Comment:
But there are too many parameters, `previousToken` `previousKind` `currentKind`, in a similar check later: `currentToken` `currentKind` `previousKind`, if cut the kind parameter for corresponding token, then another call to `.Kind()` would be introduced,  in worst would introduce 2+2 calls in total. There are code paths like `currentKind == Comma && HasFormattableBracketParent(currentToken)`, I don't think adding another check to ensure the nearby token not being OmittedArraySizeExpressionToken needs a separate method.

What do you think?